Portal/src/Datahub.Core/Model/Datahub/DatahubProjectDBContext.cs

```diff
@@ -192,6 +193,13 @@ protected override void OnModelCreating(ModelBuilder modelBuilder)
                   .WithOne(e => e.MonthlyUsage)
                   .OnDelete(DeleteBehavior.NoAction);
         });
+        modelBuilder.Entity<Project_Whitelist>(entity =>
+        {
+            entity.HasKey(e => e.Id);
+            entity.HasOne(e => e.Project)
+                .WithOne(e => e.Whitelist)
+                .OnDelete(DeleteBehavior.NoAction);
```

Comment:
Any reason we have this OnDelete behavior

Reply:
No particular reason here, I was working with @iroelperez and running into some weird migration issues. We were trying to make sure there were no differences between the project whitelist and projected credits so we could rule out anything... I think this is the default behaviour though